libs/s3/s3_test_lib.py

```diff
@@ -586,6 +586,11 @@ def create_bucket_put_object(self,
             LOGGER.debug("Creating a bucket with name %s", str(bucket_name))
             create_bucket = self.create_bucket(bucket_name)
             LOGGER.debug("Created a bucket with name %s", str(bucket_name))
+            LOGGER.info("Check bucket is empty")
+            resp = self.object_list(bucket_name)
+            if resp[1]:
+                raise Exception("Bucket is not empty %s" .format(resp[1]))
```

Comment:
We would need to raise a CTException here and not a generic Exception.

Reply:
done